test/recipe_box/cuisine_test.exs

```diff
@@ -4,26 +4,73 @@ defmodule RecipeBox.CuisineTest do
   alias RecipeBox.Cuisine
 
    describe "create_cuisine/1" do
-    @valid_attrs %{
-      name: "Italian"
-    }
+     @valid_attrs %{
+       name: "Italian"
+     }
 
-    @invalid_attrs %{}
+     @invalid_attrs %{}
 
-    test "with valid data inserts cuisine" do
-      assert {:ok, %Cuisine{id: id}=cuisine} = Cuisine.create_cuisine(@valid_attrs)
-      assert cuisine.name == "italian"
-    end
+     test "with valid data inserts cuisine" do
+       assert {:ok, %Cuisine{id: id}=cuisine} = Cuisine.create_cuisine(@valid_attrs)
+       assert cuisine.name == "italian"
+     end
 
-    test "does not insert duplicate cuisine" do
-      assert {:ok, %Cuisine{id: id}} = Cuisine.create_cuisine(@valid_attrs)
-      assert {:error, changeset} = Cuisine.create_cuisine(@valid_attrs)
-      assert errors_on(changeset) == %{name: ["has already been taken"]}
-    end
+     test "does not insert duplicate cuisine" do
+       assert {:ok, %Cuisine{id: id}} = Cuisine.create_cuisine(@valid_attrs)
+       assert {:error, changeset} = Cuisine.create_cuisine(@valid_attrs)
+       assert errors_on(changeset) == %{name: ["has already been taken"]}
+     end
 
      test "does not insert blank cuisine" do
        assert {:error, changeset} = Cuisine.create_cuisine(@invalid_attrs)
        assert errors_on(changeset) == %{name: ["can't be blank"]}
      end
+   end
+
+  describe "list_cuisines/0" do
+     @valid_attrs1 %{
+       name: "Breakfast"
+     }
+
+     @valid_attrs2 %{
+       name: "Second Breakfast"
+     }
+
+     test "returns all cuisines" do
+       %Cuisine{id: id1} = cuisine_fixture()
+       assert [%Cuisine{id: ^id1}] = Cuisine.list_cuisines() 
+
+       %Cuisine{id: id2} = cuisine_fixture(@valid_attrs1)
+       assert [%Cuisine{id: ^id1}, %Cuisine{id: ^id2}] = Cuisine.list_cuisines()
+
+       %Cuisine{id: id3} = cuisine_fixture(@valid_attrs2)
+       assert [%Cuisine{id: ^id1}, %Cuisine{id: ^id2}, %Cuisine{id: ^id3}] = Cuisine.list_cuisines()
+
+
+     end
+  end
+
+  describe "alphabetize_cuisines/1" do
+    @valid_attrs1 %{
+      name: "Breakfast"
+    }
+
+    @valid_attrs2 %{
+      name: "Second Breakfast"
+    }
+
+    @valid_attrs3 %{
+      name: "Elevenses"
+    }
+
+    test "returns cuisines in alphabetical order" do
+      %Cuisine{} = cuisine_fixture(@valid_attrs1)
+      %Cuisine{} = cuisine_fixture(@valid_attrs2)
+      %Cuisine{} = cuisine_fixture(@valid_attrs3)
+
+      cuisines = Cuisine.list_cuisines()
+
+      assert ["breakfast", "elevenses", "second breakfast"] = Cuisine.alphabetize_cuisines(cuisines)
+    end
```

Comment:
In this test you are persisting cuisines to the database, retrieving them from the database, and then calling `alphabetize_cuisines` on the returned list. Why all the ceremony up front, when this function doesn't need to know anything about a database? You could just create a list of cuisine structs and pass it directly to the function under test. Not only will that be faster, but it'll make this test less brittle—as is this test could fail if you change anything about how cuisines are saved or queried.